ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -77,6 +77,76 @@
   when: "'scaleway-ubuntu1804-armv7l' in inventory_hostname"
   include: "{{ role_path }}/tasks/partials/scaleway-armv7.yml"
 
+- name: aix install ccache - check for executable
```

Comment:
Is it possible to extract some commonality with what we do for smartos/s390x?
[`ansible/roles/baselayout/tasks/ccache.yml`](https://github.com/nodejs/build/blob/5e9585a7014ac631f2c5216dd5422e666c7636dc/ansible/roles/baselayout/tasks/ccache.yml)
https://github.com/nodejs/build/blob/06c83e72a359ff8b24e5b04bf071656f6325c2e3/ansible/roles/baselayout/vars/main.yml#L7-L10
https://github.com/nodejs/build/blob/fdd596c436764ca0d55ca210b1d16aa619e6a146/ansible/roles/baselayout/tasks/main.yml#L59-L62
https://github.com/nodejs/build/blob/fdd596c436764ca0d55ca210b1d16aa619e6a146/ansible/roles/baselayout/tasks/main.yml#L196-L206

Reply:
not 100% sure we use a different version everywhere (that seems real bad practise) and also AIX installs the ccache elsewhere. maybe that could be part of another PR to clean up how we install certain not package manager binaries?

Reply:
actually looking at `ccache.yml` its probs worth at least moving what I have to there

Reply:
reabsed and moved to `ccache.yml` and it works